pgdrive/tests/test_pgdrive_env/vis_pgdrive_env.py

```diff
@@ -28,8 +28,8 @@ def __init__(self):
                 "rgb_clip": True,
                 # "debug":True,
                 "map_config": {
-                    Map.GENERATE_METHOD: MapGenerateMethod.BIG_BLOCK_SEQUENCE,
-                    Map.GENERATE_PARA: "XTX",
+                    Map.GENERATE_METHOD: MapGenerateMethod.BIG_BLOCK_NUM,
```

Comment:
Could you make sure that y and Y will never shown up in PGDriveEnv V1 version? Could you make it part of the optional blocks in PGDrive v2?

Reply:
The prob of sampling "Merge" and "Split" are 0.00 now. I will make them only effective in PGDrive v2